test/framework/src/main/java/org/elasticsearch/index/mapper/MapperTestCase.java

```diff
@@ -1348,6 +1348,10 @@ public String parentField(String field) {
         }
     }
 
+    protected boolean nullLoaderExpected(MapperService mapper, String loaderFieldName) {
```

Comment:
If we keep it this'll need javadoc. There are tons of extensions to this class so it's super worth having something.

Reply:
I think we can remove it and move the logic into `loadBlockExpected()`